sensors:
  right_bins_camera:
    type: rgb_camera
    pose:
      xyz: [-2.286, 2.96, 1.8]
      rpy: [pi, pi/2, 0]

  left_bins_camera:
    type: rgb_camera
    pose:
      xyz: [-2.286, -2.96, 1.8]
      rpy: [pi, pi/2, 0]

  kts1_camera:
    type: rgb_camera
    pose:
      xyz: [-1.3, -5.8, 1.8]
      rpy: [pi, pi/2, pi/2]
  
  kts2_camera:
    type: rgb_camera
    pose:
      xyz: [-1.3, 5.8, 1.8]
      rpy: [pi, pi/2, -pi/2]

  right_bins_camera_logical:
    type: basic_logical_camera
    pose:
      xyz: [-2.286, 2.96, 1.8]
      rpy: [pi, pi/2, 0]

  left_bins_camera_logical:
    type: basic_logical_camera
    pose:
      xyz: [-2.286, -2.96, 1.8]
      rpy: [pi, pi/2, 0]

  kts1_camera_logical:
    type: basic_logical_camera
    pose:
      xyz: [-1.3, -5.8, 1.8]
      rpy: [pi, pi/2, pi/2]
  
  kts2_camera_logical:
    type: basic_logical_camera
    pose:
      xyz: [-1.3, 5.8, 1.8]
      rpy: [pi, pi/2, -pi/2]
  
  
  